ui/raidboss/data/07-dt/dungeon/alexandria.ts

```diff
@@ -43,10 +43,20 @@ const triggerSet: TriggerSet<Data> = {
       id: 'Alexandria AntivirusX Interferon Collect',
       type: 'AddedCombatant',
       netRegex: { name: ['Interferon C', 'Interferon R'] },
-      run: (data, matches) => {
-        const call = matches.name === 'Interferon C' ? 'Avoid' : 'In';
+      run: (data, matches, output) => {
+        const call = matches.name === 'Interferon C' ? output.avoid!() : output.in!();
         data.interferonCalls.push(call);
       },
+      outputStrings: {
+        avoid: {
+          en: 'Avoid',
+          ko: '피하기',
+        },
+        in: {
+          en: 'In',
+          ko: '안으로',
+        },
```

Comment:
Suggest changing this to `Outputs.in` (and adding `Outputs` to the import list).

Reply:
Thank you. I've updated it.